e2e/.env.test.example

```diff
@@ -0,0 +1,11 @@
+# Test database. You can use your actual MONGO_URI if you don't mind it potentially including test data.
+MONGO_URI=mongodb://127.0.0.1:27017/chatgpt-jest
+
+# Credential encryption/decryption for testing
+CREDS_KEY_PROD=c3301ad2f69681295e022fb135e92787afb6ecfeaa012a10f8bb4ddf6b669e6d
+CREDS_KEY_DEV=c3301ad2f69681295e022fb135e92787afb6ecfeaa012a10f8bb4ddf6b669e6d
+CREDS_IV_PROD=cd02538f4be2fa37aba9420b5924389f
+CREDS_IV_DEV=cd02538f4be2fa37aba9420b5924389f
```

Comment:
any chance we can change this to just CREDS_KEY & CREDS_IV? 
I'm working on removing the dev/prod versions of api keys in the user code and normally people will have different env files for different versions anyway (we could also support .env.prod too)

Reply:
Sure thing!